projection/core/src/main/scala/com/lightbend/lagom/internal/projection/ProjectionRegistryActor.scala

```diff
@@ -62,17 +65,14 @@ class ProjectionRegistryActor extends Actor with ActorLogging {
 
   val replicator: ActorRef             = DistributedData(context.system).replicator
   implicit val node: SelfUniqueAddress = DistributedData(context.system).selfUniqueAddress
-  // TODO: make timeout configurable (5.second is too aggressive in big clusters)
-  val writeMajority: WriteConsistency  = WriteMajority(timeout = 5.seconds)
-  val readConsistency: ReadConsistency = ReadLocal
 
-  // (a) Replicator contains data of all workers
-  private val RequestedStatusDataKey: LWWMapKey[WorkerKey, Status] =
-    LWWMapKey[WorkerKey, Status]("projection-registry-desired-status")
+  val writeConsistency: WriteConsistency  = WriteMajority(timeout = 5.seconds)
 
+  // (a) Replicator contains data of all workers (requested and observed status, plus a name index)
+  private val RequestedStatusDataKey: LWWMapKey[WorkerKey, Status] =
+    LWWMapKey[WorkerKey, Status]("projection-registry-requested-status")
   private val ObservedStatusDataKey: LWWMapKey[WorkerKey, Status] =
     LWWMapKey[WorkerKey, Status]("projection-registry-observed-status")
-
   private val NameIndexDataKey: LWWMapKey[WorkerKey, WorkerCoordinates] =
     LWWMapKey[WorkerKey, WorkerCoordinates]("projection-registry-name-index")
```

Comment:
The above changes (since line 65) are mostly formatting but also include:
 - renaming `writeMajority` to `writeConsistency`
 - removing the (unused) `readConsistency`
 - renaming the `projection-registry-requested-status` key for consistency.

Reply:
Maybe keep TODO comment about a configurable timeout for `writeConsistency`? 😄

Reply:
Hmm. I removed it on the grounds of https://github.com/lagom/lagom/issues/2130#issuecomment-521657624 and because there's no scenario where an `UpdateTimeout` is really an issue in this code. But I agree that a comment next to the hardcoded `5 second` will help.